src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -17,6 +17,21 @@ universes v u
 @[derive decidable_eq] inductive walking_parallel_pair : Type v
```

Comment:
Could you please add a docstring here? Like you did with the walking pair.

Reply:
Added.